src/TauBounds.h

```diff
@@ -6,8 +6,8 @@
 
 #include "LocusDataLikelihood.h"
 
-double *tau_ubounds; // Temporary array. Reinitialized per iteration. Holds upper tau bounds
-double *tau_lbounds; // Ditto, but with lower tau bounds
+extern double *tau_ubounds;
+extern double *tau_lbounds;
 
```

Comment:
Nice. BTW, if you want a global variable to be visible only in a specific module (cpp file), meaning that it's effectively private, then you do not "export" it in the header (no extern ... declaration), and in the definition in the cpp file you need to define it as static (static global variable). This ensures that it doesn't clash with another private global variable with the same name in another module. This is rarely practiced, but not bad to know :-).